app/assets/stylesheets/views/article.scss

```diff
@@ -66,8 +43,8 @@
     word-break: break-word;
 
     &__meta {
-      padding: var(--article-padding-y) var(--article-padding-x) 0
-        var(--article-padding-x);
+      padding: var(--content-padding-y) var(--content-padding-x) 0
```

Comment:
Just me generalizing names of variables...

Reply:
Love it!